spec/calendar.html

```diff
@@ -739,9 +736,9 @@ <h1>ISOMonthDayFromFields ( _fields_, _options_ )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-defaultmergecalendarfields" type="abstract operation">
+    <emu-clause id="sec-temporal-isomergecalendarfields" type="abstract operation">
       <h1>
-        DefaultMergeCalendarFields (
+        ISOMergeCalendarFields (
```

Comment:
We can remove the "fallback" language from this operation's description as well.